src/main/kotlin/no/nav/syfo/persistering/PersisterSykmeldingQueries.kt

```diff
@@ -0,0 +1,66 @@
+package no.nav.syfo.persistering
+
+import no.nav.syfo.db.Database
+import java.sql.Timestamp
+
+const val INSERT_QUERY = """
+INSERT INTO sykmelding(
+    id,
+    pasient_fnr,
+    pasient_aktoer_id,
+    lege_fnr,
+    lege_aktoer_id,
+    mottak_id,
+    legekontor_org_nr,
+    legekontor_her_id,
+    legekontor_resh_id,
+    epj_system_navn,
+    epj_system_versjon,
+    mottatt_tidspunkt,
+    sykmelding,
+    behandlings_utfall
+) VALUES (?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?)"""
+
+fun Database.insertSykmelding(sykmeldingDB: PersistedSykmelding) = connection.use { connection ->
```

Comment:
Her også burde du nok tatt inn en Sykmelding og mappet den om som en del av inserten. Men jeg vet at resten av businesslogikken bruker denne PersistedSykmelding så vi må kanskje ikke ta alt i denne PRen

Reply:
e9, jeg synes vi skal ta en db opprydding som tar i bruk interfacet dit sånn at vi får en funksjon for insert. Også vil jeg skrive om sånn at insertfunksjonen også setter opp metadata.